src/main/java/view/OutputView.java

```diff
@@ -17,20 +19,26 @@ public static void printTicketCount(int count) {
 
     public static void printTickets(List<LottoTicket> tickets) {
         for (LottoTicket ticket : tickets) {
-            System.out.println(ticket);
+            System.out.println(formatTicket(ticket));
         }
         System.out.println();
     }
 
-    public static void printResult(Map<Integer, Integer> matchResults) {
+    private static String formatTicket(LottoTicket ticket) {
+        return "[" + ticket.getNumbers().stream()
+            .map(String::valueOf)
+            .collect(Collectors.joining(", ")) + "]";
```

Comment:
`Collectors::joining` 메서드에는 prefix와 suffix를 붙일 수 있답니다.

```suggestion
        return ticket.getNumbers().stream()
            .map(String::valueOf)
            .collect(Collectors.joining(", ", "[", "]"));
```

팁으로 맥북 기준 `CMD` + `P`를 눌러 메서드의 매개변수로 어떤게 들어갈 수 있는지 확인할 수 있어요.

Reply:
아하 적용했더니 훨씬 깔끔해졌어요! 매개변수도 앞으로 자주 확인보면 좋을 것 같아요!